install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -347,6 +354,47 @@ storage that are registered with it. Trident itself is configured separately.
 <1> For more information about installing Trident with {product-title}, see the link:https://github.com/NetApp/trident[Trident documentation].
 <2> For more information about supported parameters, see the link:https://github.com/NetApp/trident#storage-attributes[storage attributes] section of the Trident documentation.
 
+[[change-default-storage-class]]
+== Changing the default StorageClass
+Use the following process to change the default StorageClass:
+
+. List the StorageClass:
++
+====
+----
+$ oc get storageclass
+
+NAME                 TYPE
+fast (default)   kubernetes.io/glusterfs <1>
```

Comment:
The default storageclasses in GCE is named `standard` and on AWS it is named `gp2`.  It might be better to use those default classes as an example so as users can relate what default classes they will have in their cluster.